src/styles/blocks/card.scss

```
@@ -0,0 +1,136 @@
+.container {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, $card-width);
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    grid-template-columns: repeat(4, $card-width);
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid $card-border;
+  background: $white;
+  padding: 32px 16px 16px;
+  transition: all 0.3s ease;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__img {
+    display: flex;
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    transition: all 0.3s ease;
+  }
+
+  &__product-code {
+    color: $secondary;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+    align-self: flex-start;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__review-count {
+    color: $main-accent;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    color: $secondary;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-number {
+    color: $main-accent;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
```

Let's fix it everywhere
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin-bottom: 16px;
    
    &-text {}
    
    &-number {}
  }
```